File: src/main/java/it/contrader/dto/UnitaMisuraDTO.java
package it.contrader.dto;

public class UnitaMisuraDTO {
	private int id;
	private String nome;
	
	public UnitaMisuraDTO() {
		
	}
	
	public UnitaMisuraDTO(String nome) {
		this.nome = nome;
	}
	
	public UnitaMisuraDTO(int id, String nome) {
		this.id = id;
		this.nome = nome;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	@Override
	public String toString() {
		return id +"\t  " + nome;
	}
	
}

File: src/dto/unitamisura.dto.ts
export class UnitaMisuraDTO{

  id: number;
  nome: string;
  categoriaId: number;

  constructor(id: number, nome: string, categoriaId: number){
    this.id = id;
    this.nome = nome;
    this.categoriaId = categoriaId;
  }

}
